### PR TITLE
Removed paragraph about docuwiki migration

### PR DESCRIPTION
**Pull request by VoyTechnology**
The docs site reflects the latest changes from the repository, no longer being hosted by docuwiki.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 Documentation for [Redbrick](https://redbrick.dcu.ie)
 
-Migrated from [dokuwiki](https://docs.redbrick.dcu.ie) to [readthedocs](http://redbrick.readthedocs.io)
+Available at https://docs.redbrick.dcu.ie and at the (readthedocs mirror)[https://redbrick.readthedocs.io]
```
Comment (butlerx) [inline README.md]: make this a link

**Review by butlerx — CHANGES_REQUESTED** [review]

**Comment by VoyTechnology** [comment]
@butlerx please take another look

**Review by butlerx — APPROVED** [review]

**Comment by SeanHealy33** [comment]
> Merge branch 'master' into remove-migration-notice

The no-rebase trigger warning.

**Comment by VoyTechnology** [comment]
@SeanHealy33, it gets squashed anyways, and personally I prefer to have merge commits so I keep track of when I merged etc.

**Pull request merged**